sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -1,30 +1,12 @@
-# Azure Text Analytics client library for JavaScript
+# Azure Text Analytics Client library for JavaScript
 
-[Azure TextAnalytics](https://azure.microsoft.com/services/cognitive-services/text-analytics/) is a cloud-based service that provides advanced natural language processing over raw text, and includes six main functions:
+This package contains an isomorphic SDK for Azure Text Analytics Client.
 
-**Note:** This SDK targets Azure Text Analytics service API version 3.1.0-preview.3.
+The Text Analytics API is a suite of natural language processing (NLP)  services built with best-in-class Microsoft machine learning algorithms.  The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction and language detection. Functionality for analysis of text specific to the healthcare domain and personal information are also available in the API. Further documentation can be found in <a href="https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview">https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview</a>
```

Comment:
Where is this text coming from? The swagger? Or the readme file which has the metadata?

Reply:
The swagger.